### ISSUE TITLE
Accessibility: Status message not announced by screen readers [Product Single (All Types)]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51931]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The add to cart success message is visually displayed, but screen readers do not announce it (though it has a role of alert) because the page refreshes and screen readers instead read the page title/top of the page.

![Image](https://github.com/user-attachments/assets/3d6b06ba-a674-4a81-a96f-37df393232e7)

### Component or Page: 

Page: Product Single (All types)  

### Name affected element:

Success message for adding product to cart

### Relevant code snippets:

```
<div class=""woocommerce-notices-wrapper"">
	<div class=""wc-block-components-notice-banner is-success"" role=""alert"">
		<svg xmlns=""http://www.w3.org/2000/svg"" viewBox=""0 0 24 24"" width=""24"" height=""24"" aria-hidden=""true"" focusable=""false"">
			<path d=""M16.7 7.1l-6.3 8.5-3.3-2.5-.9 1.2 4.5 3.4L17.9 8z""></path>
		</svg>
		<div class=""wc-block-components-notice-banner__content"">
			<a href=""https://wooprd.wpenginepowered.com/cart/"" tabindex=""1"" class=""button wc-forward wp-element-button"">View cart</a> “Intelligent Paper Knife” and “Durable Granite Knife” have been added to your cart.		</div>
	</div>
</div>
```

### Recommended fix:

Possible options to remediate this:

A) Don't refresh the page on submit of the ""Add to Cart"" button and instead use JavaScript to triggert the action/show the error.

B) After the page is refreshed, use JavaScript to set keyboard focus on the success message so screen readers read it rather than the content at the top of the page.

This is connected to https://github.com/woocommerce/woocommerce/issues/51929

**Comment by Manussakis** [comment]
The issue title reads "Product Single" but the screenshot in the description is of a Group Product.

I was not able to reproduce this issue on a Single Product page. Most of the issues with alert messages were fixed once the https://github.com/woocommerce/woocommerce/pull/50061 PR was merged.

The screenshot below shows the alert message being focused and the screen reader announcing it after adding the product to the cart.
![Image](https://github.com/user-attachments/assets/e3602f14-29c4-48c6-af3b-064b72daf098)

Unless I missed something, this issue can be closed.

cc: @vikrampm1

**Comment by Manussakis** [comment]
@vikrampm1 I linked the https://github.com/woocommerce/woocommerce/pull/50061 PR with this issue.
